ospd_openvas/daemon.py

```diff
@@ -1017,11 +1082,9 @@ def stop_scan_cleanup(
                 self.openvas_db.release_db(current_kbi)
                 for host_kb in range(0, self.openvas_db.max_dbindex):
                     self.openvas_db.select_kb(
-                        ctx,
```

Comment:
These formatting changes would have been better in a separate commit.

Reply:
I did not realize about this since it is done automatically when I commit. There is a hook which runs black.